hw/ip/otbn/dv/model/iss_wrapper.cc

```diff
@@ -355,6 +355,24 @@ void ISSWrapper::edn_urnd_step(uint32_t edn_urnd_data) {
   run_command(oss.str(), nullptr);
 }
 
+void ISSWrapper::set_keymgr_value(std::array<uint32_t, 12> key0_arr,
+                                  std::array<uint32_t, 12> key1_arr,
+                                  unsigned char valid) {
```

Comment:
I think I'd suggest changing this function to take a boolean. The `unsigned char` is because that's how DPI-C represents a SystemVerilog `bit`, but we should probably convert to a more convenient type at the same time as we shove stuff in the `std::array`.

Reply:
Changed unsigned_char to bool